doc/extension.rst

```diff
@@ -121,6 +81,14 @@ The extension has a few configuration options that can be set in ``conf.py``:
    You can also pass in the compiler to use, for example
    ``get_include_args('gcc')``.
 
+.. py:data:: cautodoc_transformations
+   :type: dict
+   :noindex:
+
+   This configuration option and its functionality has been moved to the
+   built-in :ref:`hawkmoth.ext.transformations` extension. See
+   :py:data:`cautodoc_transformations`.
```

Comment:
On a similar note, is this supposed to be a reference to itself?

Reply:
Same here, with `:noindex:` this should be a reference to the definition in `built-in-extensions.rst`.

Reply:
Resolved in #131 by just adding a reference to configuration in built-in extensions.